Source/WebCore/platform/graphics/texmap/BitmapTexture.cpp

```diff
@@ -86,7 +86,11 @@ void BitmapTexture::updateContents(const void* srcData, const IntRect& targetRec
     // We are updating a texture with format RGBA with content from a buffer that has BGRA format. Instead of turning BGRA
     // into RGBA and then uploading it, we upload it as is. This causes the texture format to be RGBA but the content to be BGRA,
     // so we mark the texture to convert the colors when painting the texture.
+#if G_BYTE_ORDER == G_LITTLE_ENDIAN
```

Comment:
Looks like this doesn't build on Windows. This file is cross-platform and shouldn't be using GLib stuff.

Reply:
Yes, I just saw this, too.
The question now is which of the various ways to detect endianness used in WebKit should I use in this file?

Reply:
Let's see if Windows has `__BYTE_ORDER__`…

Reply:
Seems that worked.